deployment_jobs/Create_Model_and_Deployment_Job.py

```diff
@@ -20,16 +21,6 @@
 
 # COMMAND ----------
 
-# Create registered model
```

Comment:
is this moved on purpose?

Reply:
Yeah we never updated it to add the deployment job ID before